can-bind.js

```diff
@@ -25,8 +27,10 @@ function defaultSetValue(newValue, observable) {
 
 // Given an observable, stop listening to it and tear down the mutation dependencies
 function turnOffListeningAndUpdate(listenToObservable, updateObservable, updateFunction, queue) {
-	if (listenToObservable[onValueSymbol]) {
-		canReflect.offValue(listenToObservable, updateFunction, queue);
+	var offValueOrOffEmitFn = getOffValueOrOffEmitFunctionFromObservable(listenToObservable);
```

Comment:
Is there a reason for making the `getOffValueOrOffEmitFunctionFromObservable` function, which seems to only be used in this one place? Same thing with `getOnValueOrOnEmitFunctionFromObservable `.

Reply:
It could be done inside the `turnOn` / `turnOff` functions, I thought it read better to break it out, there was more logic in the `getOnValueOrOnEmitFunctionFromObservable` functions but they have since been simplified so I'll move it inside of the `turnOffListeningAndUpdate` and `turnOnListeningAndUpdate` methods.